cmd/server/server.go

```diff
@@ -111,7 +112,11 @@ func (s *server) startService() common.Daemon {
 		log.Fatalf("error creating ringpop factory: %v", err)
 	}
 
-	params.DynamicConfig = dynamicconfig.NewNopClient()
+	params.DynamicConfig, err = dynamicconfig.NewFileBasedClient(&s.cfg.DCClientConfig, params.Logger.WithTags(tag.Service(params.Name)), s.doneC)
+	if err != nil {
+		log.Printf("error creating file based dynamic config client: %v", err)
```

Comment:
I would make it:
"cannot create filed based dynamic config client, so will use no-op config client instead. error: %v"

Reply:
Done.